### PR TITLE
Human player strategy

### PR DESCRIPTION
**Pull request by jun-loke**
lotsa user input... so extremely much user input

### CODE REVIEW

**Review by felixlapierre — APPROVED** [review]
Tested it out on my end and everything looks great :D

**Comment by johnstonstott** [comment]
if you put user input that isn't numeric it infinite loops 
image 

but tbh don't think its a big issue

**Comment by johnstonstott** [comment]
Besides this looks s t e l l a r

File: warzone/TEMPFILE.cpp [hunk 1]
```diff
@@ -0,0 +1,244 @@
+#include "PlayerStrategies.h";
```
Comment (johnstonstott) [inline warzone/TEMPFILE.cpp]: What is this file?

Comment (jun-loke) [reply warzone/TEMPFILE.cpp]: I hate that they make us have all the strategies in a single file so when i work on it, i had all the HumanStrategy function definitions in here.

**Comment by jun-loke** [comment]
> if you put user input that isn't numeric it infinite loops
> image
> 
> but tbh don't think its a big issue

Fixed in f084758

**Comment by johnstonstott** [comment]
Maybe I entered an invalid state or something but it doesn't like 'y'

image

Or any input for that matter

**Review by johnstonstott — APPROVED** [review]
Seems to work well now! Nice job

**Pull request merged**